cmake/std/atdm/ctest-s-local-test-driver.sh

```diff
@@ -301,7 +301,7 @@ for build_name_body in ${ATDM_ARRAY_OF_BUILDS[@]} ; do
     &> smart-jenkins-driver.out
 
   echo
-  grep "failed out of" smart-jenkins-driver.out
+  grep "failed.*out.*of" smart-jenkins-driver.out
```

Comment:
Was this `grep` missing failures or did the contents of smart-jenkins-driver.out change?

Reply:
> Was this grep missing failures or did the contents of smart-jenkins-driver.out change?

@e10harvey, with the updated CTest 3.16 using color output, it was missing the grep of any test results.  This works now with all versions of CTest, with or without color.